source/icu.net.tests/Collation/CollatorTests.cs

```diff
@@ -100,7 +100,7 @@ public void Create_NoPredefinedCollator_Throws()
 		{
 			Assert.That(() =>
 			{
-				using (Collator.Create("my-MM")) {}
+				using (Collator.Create("my-MM")) { }
```

Comment:
It doesn't appear as though we have anything in .editorconfig that suggests whether these spaces are desirable. I assume you're using some tool that has a preference for this. I don't much care one way or the other, but unless the standard is documented and/or enforced by tooling, if someone else edits this file and uses some other tool to enforce consistency, it could easily go back the other way.

Reply:
Yeah, that's my autoformatter. I don't mind if others' revert it.

Reply:
> Yeah, that's my autoformatter. I don't mind if others' revert it.

The problem is that **Blame** becomes much more tedious to use if spurious whitespace changes hide who the actual author of a line of code was. (Obviously, there's always going to be some such obfuscation, but it'd be nice to avoid in cases where there is no agreed-upon preference.)

Reply:
For the sake of reviewers and posterity, I've retracted the majority of those unnecessary changes.